tools/build_langserver/lsp/symbols_test.go

```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-const testURI = "file://test/test.build"
+const testURI = "file:///test_data/test.build"
```

Comment:
This doesn't materially affect anything, but does remove the confusing log line about relative paths not being resolved.